CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ IMPROVEMENTS:
 * resource/nomad_csi_volume: changes to `capacity_min` or `capacity_max` may now expand the volume instead of forcing replacement,
   on Nomad version 1.6.3 or later, if the CSI plugin supports it ([#382](https://github.com/hashicorp/terraform-provider-nomad/pull/382))
 * resource/acl_auth_method: add support for `token_name_format` ([#403](https://github.com/hashicorp/terraform-provider-nomad/pull/403))
+* resource/nomad_job: add support for importing jobs from non-default namespace ([#408](https://github.com/hashicorp/terraform-provider-nomad/pull/408))
```

Comment:
This is a little misleading -- to me it implies no changes if you always use the default namespace, but we're actually making it _mandatory_ to include the namespace when importing these resources, even when it's `@default` (due to the challenging fact that job IDs can contain just about any character, including `@`).  Nice to lean positive with "add support" but I might also mention the new requirement.

Also, is it worth calling CSI volumes here too? I imagine they're much less often used, so maybe ok to leave out of the CL.

Reply:
I has always been possible to import resources from the `default` namespace, so the change is that non-default namespaces are now supported. But I see what you mean. Updated to:

> * resource/nomad_job: update import key to be `<job id>@<namespace>` to allow importing jobs from namespaces other than `default` ([#408](https://github.com/hashicorp/terraform-provider-nomad/pull/408))


Also added similar entries to volumes. I just forgot about them 😅